contrib/pyln-testing/requirements.txt

```diff
@@ -1,6 +1,7 @@
-pytest==5.0.1
+pytest==5.3.1
 Flask==1.1.1
-cheroot==6.5.5
+cheroot==8.2.1
 ephemeral-port-reserve==1.1.1
 python-bitcoinlib==0.10.1
```

Comment:
Trailing old requirement?

Reply:
good question, I think this was just left over when reorganizing from pylightning into pyln.

Reply:
we could/should update `contrib/Dockerfile.builder` and `contrib/Dockerfile.builder.i386` as well they also point to that old version.

Reply:
I was talking about `python-bitcoinlib`